abb_experimental/package.xml

```diff
@@ -8,9 +9,13 @@
   <author email="sedwards@swri.org">Shaun Edwards</author>
 
   <buildtool_depend>catkin</buildtool_depend>
-  <run_depend>abb_irb4400_support</run_depend>
+
+  <exec_depend>abb_resources</exec_depend>
+  <exec_depend>abb_driver</exec_depend>
+  <exec_depend>abb_irb4400_support</exec_depend>
```

Comment:
Did you have a particular reason to include only the `abb_irb4400_support` package here? And not the ones for the IRB 120?

Also: `abb_resources` and `abb_driver` are packages in another repository. Did you add those intentionally?